icevision/core/mask.py

```diff
@@ -64,10 +64,11 @@ class MaskArray(Mask):
 
     # Arguments
         data: Mask array, with the dimensions: (num_instances, height, width)
+        pad_dim: bool
     """
 
-    def __init__(self, data: np.uint8):
-        if len(data.shape) == 2:
+    def __init__(self, data: np.uint8, pad_dim: bool = True):
+        if pad_dim and (len(data.shape) == 2):
```

Comment:
What is `pad_dim` for?

Reply:
This change introduced the option **not to** pad the mask dimensions. Before this change, the mask array always ended up being padded.

Reply:
During the transforms?